packages/framework-provider-aws/src/library/read-models-adapter.ts

```diff
@@ -141,3 +154,11 @@ function buildKey(
   if (sequenceKeyName && sequenceKeyValue) return { id: readModelID, [sequenceKeyName]: sequenceKeyValue }
   else return { id: readModelID }
 }
+
+function generateOptimisticConcurrencyValue(value: ReadModelInterface): number {
+  let optimisticConcurrencyValue = 1 // if there is not version then we need to persist the first one
+  if (value.boosterMetadata?.version) {
+    optimisticConcurrencyValue = value.boosterMetadata?.version + 1 // the next version number that we are going to persist
+  }
+  return optimisticConcurrencyValue
```

Comment:
The Rubyist inside of me yields this :trollface: (totally optional, the clearer the better!)

```suggestion
  return (value.boosterMetadata?.version ?? 0) + 1 // the next version number that we are going to persist
```

Reply:
I'd say that this one is better because its more idiomatic + we avoid having a `let` which is mostly always bad :)